_maps/WariosBattleCanyon/Wario's Battle Canyon.yaml

```diff
@@ -0,0 +1,177 @@
+---
+name:
+  en: Wario's Battle Canyon
+desc:
+  en: Wario's Battle Canyon from Mario Party 1. Use the warps and cannons to jump between the islands, hit the switch to change the order of the warps!
+ruleSet: Standard
+theme: Mario
+initialCash: 3000
+targetAmount: 20000
+baseSalary: 300
+salaryIncrement: 200
+maxDiceRoll: 8
+frbFile1: WariosBattleCanyon0
+frbFile2: WariosBattleCanyon1
+tourMode:
+  bankruptcyLimit: 1
+  opponent1: Mario
+  opponent2: Slime
+  opponent3: BowserJr
+  clearRank: 1
+background: battlecanyon
+mapIcon: battlecanyon
+music:
+  download: https://drive.google.com/u/0/uc?id=1eUv7IEt9dzQr98gVMQfSb_CnSDzB8sHG&export=download
```

Comment:
this download URL is not needed here, since it has already been defined in backgrounds.yml.